environment.yml

```diff
@@ -1,12 +1,13 @@
 name: root
 dependencies:
 - beautifulsoup4=4.5.1
-- boto=2.43.0
-- cython=0.25.1
+- boto=2.45.0
+- cython=0.25.2
 - ipython=5.1.0
+- jinja2=2.8
+- joblib=0.9.4
```

Comment:
Sorry, I missed that this hadn't changed. Could you use joblib v0.10.3? I'd rather not have it in the environment at all than to have v0.9.